common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -707,7 +711,7 @@ def mock_val_export(edx_video_id, course_id):
            <handout src="http://www.example.com/handout"/>
            <transcript language="ge" src="german_translation.srt" />
            <transcript language="ua" src="ukrainian_translation.srt" />
-           <video_asset export_edx_video_id="test_edx_video_id"/>
+           <video_asset video_ids="[u'test_edx_video_id', u'p2Q6BrNhdh8', 'source', 'source']"/>
```

Comment:
this is confusing, can we just verify parameters with `assert_called_with` and remove this edx-val's mocked xml?